get_mozilla_keys.sh

```diff
@@ -46,4 +47,7 @@ echo
 get_key "dep1.der" "dep1_sha1" $SHA1_REV
 echo
 get_key "dep2.der" "dep2_sha1" $SHA1_REV
+echo
+
+get_key "autograph_stage.der" "autograph_stage_sha384" $SHA384_REV
```

Comment:
We need to change this to `autograph_stage.pem`, and possibly write a new `get_pem_key` helper function to not expect a `der` format file.